SimTracker/SiPhase2Digitizer/plugins/PSPDigitizerAlgorithm.cc

```diff
@@ -52,3 +57,17 @@ bool PSPDigitizerAlgorithm::isAboveThreshold(const DigitizerUtility::SimHitInfo*
                                              float thr) const {
   return (charge >= thr);
 }
+//
+//  Check whether the Hit is in the Inefficient Bias Rail Region
+//
+bool PSPDigitizerAlgorithm::isInBiasRailRegion(const PSimHit& hit) const {
+  static const float implant = 0.1467;  // Implant length (1.467 mm)
+  static const float bRail = 0.00375;   // Bias Rail region which causes inefficiency (37.5micron)
+  float block_len = 16 * implant + 15.5 * bRail;
```

Comment:
This could be `static const` too (or even better `constexpr`, this applies to the two above too).

Reply:
for future reference, maybe you can also put a comment on what are `16` and `15.5`?